fsharp-backend/src/LibBackend/Pusher.fs

```diff
@@ -69,10 +72,19 @@ let pushNew404 (canvasID : CanvasID) (f404 : TraceInputs.F404) =
 let pushNewStaticDeploy (canvasID : CanvasID) (asset : StaticAssets.StaticDeploy) =
   push canvasID "new_static_deploy" (Json.Vanilla.serialize asset)
 
+type AddOpEventTooBigPayload = { tlids : List<tlid> }
 
 // For exposure as a DarkInternal function
 let pushAddOpEvent (canvasID : CanvasID) (event : Op.AddOpEvent) =
-  push canvasID "add_op" (Json.OCamlCompatible.serialize event)
+  let payload = Json.OCamlCompatible.serialize event
+  if String.length payload > 10240 then
+    let tlids = List.map LibExecution.OCamlTypes.tlidOf event.``params``.ops
+    let tooBigPayload = { tlids = tlids } |> Json.Vanilla.serialize
+    // push canvasID "addOpTooBig" tooBigPayload
```

Comment:
do you need to uncomment this?

Reply:
No, that was the point of the comment that immediately follows.

I guess I should make that clearer?

Reply:
My interpretation of the line is "push a notification that there was some add_op involving so many canvases that the serialized payload would be huge." That seems good+safe to have even with no receiver? (i.e. so we can use that data for some analysis, if pushes are persisted?)

Reply:
I think it'll make a BIG RED ERROR BAR with no receiver, which the user can't do anything about.

I'll fix the comment.

Reply:
Ah, ok. Maybe we should (separately) _not_ raise a big red error bar when there's no receiver?
That sounds like a reasonable way to tee up something to be handled later, recording along the way for potential analysis.

Reply:
PR looks good with comment update.

Reply:
Thanks